worldedit-bukkit/src/main/java/com/fastasyncworldedit/bukkit/FaweBukkit.java

```diff
@@ -299,11 +300,8 @@ private void setupPlotSquared() {
         if (plotSquared == null) {
             return;
         }
-        if (plotSquared.getClass().getPackage().toString().contains("intellectualsites")) {
-            WEManager.IMP.managers.add(new com.fastasyncworldedit.bukkit.regions.plotsquaredv4.PlotSquaredFeature());
-            LOGGER.info("Plugin 'PlotSquared' found. Using it now.");
-        } else if (PlotSquared.get().getVersion().version[0] == 6) {
-            WEManager.IMP.managers.add(new com.fastasyncworldedit.bukkit.regions.plotsquared.PlotSquaredFeature());
+        if (PlotSquared.get().getVersion().version[0] == 6) {
+            WEManager.IMP.managers.add(new PlotSquaredFeature());
             LOGGER.info("Plugin 'PlotSquared' found. Using it now.");
         } else {
             LOGGER.error("Incompatible version of PlotSquared found. Please use PlotSquared v6.");
```

Comment:
```suggestion
            LOGGER.error("Incompatible version of PlotSquared found. Please use PlotSquared v6.");
            LOGGER.info("https://www.spigotmc.org/resources/77506/.");
```
v6 is a different resource than v4/older releases. It can't hurt to direct people to the proper download spot.